### PR TITLE
Fix profile distance measurement

### PR DESCRIPTION
**Pull request by oterral**
[Test](https://mf-geoadmin3.int.bgdi.ch/teo_fixes/88bcc8a/1472629233/index.html)

@loicgasser quick review please

### CODE REVIEW

**Comment by cedricmoullet** [comment]
Not related to this fix, but when measuring in a lake, the profile looks like empty and not flat:
image

**Comment by gjn** [comment]
Same height everywhere? Has been like that for long time. Not related to draw V2.

**Comment by cedricmoullet** [comment]
@davidoesch please decide if it's a bug

**Comment by davidoesch** [comment]
not a bug it is a feature: lake is flat at 429m

File: src/components/profile/ProfileService.js [hunk 1]
```diff
@@ -133,7 +133,6 @@ goog.require('ga_urlutils_service');
             var denom = maxX >= 10000 ? 1000 : 1;
```
Comment (loicgasser) [inline src/components/profile/ProfileService.js]: not using it anymore

**Comment by loicgasser** [comment]
one comment LGTM otherwise.

**Pull request merged**